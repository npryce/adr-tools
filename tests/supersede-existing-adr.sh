adr new First Record
adr new -s 1 Second Record
head -10 doc/adr/0001-first-record.md
head -12 doc/adr/0002-second-record.md

