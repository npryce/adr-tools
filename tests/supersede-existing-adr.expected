adr new First Record
doc/adr/0001-first-record.md
adr new -s 1 Second Record
doc/adr/0002-second-record.md
head -10 doc/adr/0001-first-record.md
# 1. First Record

Date: 1992-01-12

## Status

Superseded by [2. Second Record](0002-second-record.md)

## Context

head -12 doc/adr/0002-second-record.md
# 2. Second Record

Date: 1992-01-12

## Status

Accepted

Supersedes [1. First Record](0001-first-record.md)

## Context


